packages/truffle-core/lib/commands/obtain.js

```diff
@@ -10,6 +10,11 @@ module.exports = {
           `Download and cache a version of the specified compiler.\n` +
           `                    compiler_name must be one of the following: ` +
           `'solc'.(required)`
+      },
+      {
```

Comment:
Instead of adding a new option, edit the existing one above. Change `<--<compiler_name> <version>>` to `--solc <version>`. Or just delete the first option and leave the one you added.

Reply:
I think I might find the problem of low code coverage. I am testing the build to see the coverage now.